src/psyclone/psyir/symbols/symboltable.py

```diff
@@ -230,6 +230,29 @@ def _validate_non_args(self):
                         "yet has an ArgumentInterface interface."
                         "".format(str(symbol)))
 
+    def get_unlinked_datasymbols(self, ignore_precision=True):
+        '''
+        Create a list of the names of all of the Symbols in the table that
```

Comment:
Replace "Symbols" with "Data Symbols" in this description?

Reply:
Done.